proxy/envoy/route.go

```diff
@@ -31,10 +31,10 @@ import (
 
 const (
 	// InboundClusterPrefix is the prefix for service clusters co-hosted on the proxy instance
-	InboundClusterPrefix = "in:"
+	InboundClusterPrefix = "in."
```

Comment:
Why `.` and not `|` here? 

Also, does it make sense to parameterize the separator? e.g.

```golang
const ServiceSeperator = "|"
```